populus/compilation/backends/vyper.py

```diff
@@ -12,7 +12,7 @@ class VyperBackend(BaseCompilerBackend):
 
     def get_compiled_contracts(self, source_file_paths, import_remappings):
         try:
-            from vyper import compiler
+            from .vyper import compiler
```

Comment:
This seemed wrong to me and I've fixed this in #469. Although `vyper` library and this module have the same name, import inside the function works fine.